hack/jenkins/common.sh

```diff
@@ -41,8 +43,8 @@ sudo ./installers/check_install_golang.sh "1.15.5" "/usr/local" || true
 # install docker and kubectl if not present
 sudo ./installers/check_install_docker.sh
 
-docker rm -f -v $(docker ps -aq) >/dev/null 2>&1 || true
-docker volume prune -f || true
+# let's just clean all docker artifacts up
+docker system prune --all --force --volumes || true
```

Comment:
does this remove all the cached docker  images too ? 
in my experinece this was adding a lot trouble with docker rate limit and had to redownload kicbase images again and again each single time it runs..


but if --volumes (makes it not delete images  I m okay with this)

Reply:
this does delete images, but that was the point. there were a couple of debian agents that got messed up because there were SO many images in docker that it was causing failures for the none driver.

Reply:
there is also one option that deletes only ancient images that one is better

Reply:
ok switched it to only delete dangling images